src/pages/admin/AdminContestsList/CreateContestModal.tsx

```diff
@@ -0,0 +1,402 @@
+      const endDate = startDate.plus({ hours: 24 * parseInt(contestAuditLength) })
+      const judgingEndDate = endDate.plus({ hours: 24 * 3 })
+
+      setContestJudgingContestEndDate(judgingEndDate.toFormat(DATE_FORMAT))
+    }
+  }, [contestStartDate, contestAuditLength, setContestJudgingContestEndDate])
+
+  useEffect(() => {
+    if (isSuccess) onClose?.()
+  }, [isSuccess, onClose])
+
+  useEffect(() => {
+    if (twitterAccount?.profilePictureUrl) {
+      setProtocolLogoURL(twitterAccount.profilePictureUrl)
+    } else {
+      setProtocolLogoURL("")
+    }
+  }, [twitterAccount])
+
+  useEffect(() => {
+    if (contestStartDate === "") {
+      setStartDateError(undefined)
+      return
+    }
+
+    const startDate = DateTime.fromFormat(contestStartDate, DATE_FORMAT)
+
+    if (!startDate.isValid) {
+      setStartDateError(`Invalid date. Must be format ${DATE_FORMAT}`)
+      return
+    }
+
+    updateDates()
+
+    if (startDate < DateTime.now()) {
+      setStartDateError("Start date cannot be in the past.")
+      return
+    }
+
+    setStartDateError(undefined)
+  }, [contestStartDate, updateDates, contestAuditLength])
+
+  useEffect(() => {
+    if (!contestAuditPrizePool) return
+
+    const fixedPay = contestAuditPrizePool.div(10).mul(4)
+    const judgingPool = contestAuditPrizePool.div(100).mul(5)
+
+    setInitialLeadSeniorWatsonFixedPay(fixedPay)
+    setInitialJudgingPrizePool(judgingPool)
+  }, [contestAuditPrizePool, setInitialLeadSeniorWatsonFixedPay, setInitialJudgingPrizePool, setInitialTotalCost])
+
+  useEffect(() => {
+    const total = contestAuditPrizePool
+      ?.add(contestLeadSeniorWatsonFixedPay ?? 0)
+      .add(contestJudgingPrizePool ?? 0)
+      .add(contestAuditPrizePool.div(100).mul(5))
+    setInitialTotalCost(total)
+  }, [contestAuditPrizePool, contestLeadSeniorWatsonFixedPay, contestJudgingPrizePool])
+
+  const sherlockFee = useMemo(() => {
+    const fee = contestTotalCost
+      ?.sub(contestAuditPrizePool ?? 0)
+      .sub(contestLeadSeniorWatsonFixedPay ?? 0)
+      .sub(contestJudgingPrizePool ?? 0)
+
+    return commify(parseInt(ethers.utils.formatUnits(fee ?? 0, 6)))
+  }, [contestAuditPrizePool, contestLeadSeniorWatsonFixedPay, contestJudgingPrizePool, contestTotalCost])
+
+  const canCreateContest = useMemo(() => {
+    if (protocolName === "") return false
+    if (protocolLogoURL === "" && !protocol?.logoURL) return false
+    if (protocolWebsite === "" && !protocol?.website) return false
+    if (protocolTwitter === "" && !protocol?.twitter) return false
+
+    if (contestTitle === "") return false
+    if (contestShortDescription.length < 100 || contestShortDescription.length > 200) return false
+
+    if (contestAuditLength === "") return false
+
+    const startDate = DateTime.fromFormat(contestStartDate, DATE_FORMAT)
+    const endDate = startDate.plus({ hours: 24 * parseInt(contestAuditLength) })
+    const judgingEndDate = endDate.plus({ hours: 24 * 3 })
```

Comment:
Shouldn't we check here the `contestJudgingContestEndDate` instead of `endDate.plus({ hours: 24 * 3 })`?

Reply:
yeah, great catch!